### PR TITLE
40K: other Ravenous card scripts

### PR DESCRIPTION
**Pull request by Northmoc**

### CODE REVIEW

**Comment by Northmoc** [comment]
needs #1507 before merging

File: forge-gui/res/cardsfolder/upcoming/ravener.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Name:Ravener
+ManaCost:X G U
+Types:Creature Tyranid
+PT:0/0
+K:Flash
+K:Ravenous
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigFight | TriggerDescription$ Terror from the Deep — When CARDNAME enters the battlefield, target creature attacks target opponent this turn if able.
+A:AB$ Pump | ValidTgts$ Creature | SubAbility$ DBEffect
+SVar:DBEffect:DB$ Effect | ValidTgts$ Opponent | RememberObjects$ ParentTarget,Targeted | StaticAbilities$ MustAttack
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/ravener.txt:9]: Add `ExileOnMoved$ Battlefield` to help keep GUI/engine clean

Comment (Northmoc) [reply forge-gui/res/cardsfolder/upcoming/ravener.txt:9]: Wouldn't the effect still hold true for this turn if Ravener was removed?

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/ravener.txt:9]: It will trigger on the remembered objects of the effect

**Comment by tool4ever** [comment]
Try rebasing for the checks

File: forge-gui/res/cardsfolder/upcoming/tervigon.txt [hunk 2]
```diff
@@ -0,0 +1,12 @@
+NameTervigon
+ManaCost:X 1 G
+Types:Creature Tyranid
+PT:0/0
+K:Ravenous
+K:Trample
+T:Mode$ DamageDone | ValidSource$ Card.Self | ValidTarget$ Player | CombatDamage$ True | Execute$ TrigToken | TriggerDescription$ Spawn Termagants — Whenever CARDNAME deals combat damage to a player, create that many 1/1 green Tyranid creature tokens.
+SVar:TrigToken:DB$ Token | TokenAmount$ Y | TokenScript$ g_1_1_tyranid
+SVar:X:Count$xPaid
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/tervigon.txt:9]: X SVar is unneeded but that shouldn't be the real issue, hmn

**Pull request merged**